CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 * Added the pack version to the code files when calling **unify**. The same value is removed when calling **split**.
+* Added support in the `<~XPANSE>` marketplace tag for release notes.
```

Comment:
```suggestion
* Added support for the `<~XPANSE>` marketplace tag on **update-release-notes**.
```

Reply:
It's not for update-release-notes, it's for the release notes generated by this command - but I'll make a combination of this and your suggestion 👍🏽